docs/src/examples/scikit-learn.rst

```diff
@@ -0,0 +1,112 @@
+user. Oríon solves this problem by providing cutting-edge hyper-parameter optimization
+algorithms without disrupting the workflow of the user. Integrating it only require minimal
+adjustments to your current workflow as we'll demonstrate in the next section.
+
+Enter Orion
+-----------
+Integrating Oríon into your workflow requires only two non-invasive changes:
+   1. Define an objective to optimize.
+   2. Specify the hyper-parameter space.
+
+For the former, this step takes place in the script training the model. The latter can either be
+specified in a configuration file or directly while calling the script with Oríon.
+For the purpose of the example, we'll configure the hyper-parameter space directly as a
+command-line argument.
+
+Updating the script
+^^^^^^^^^^^^^^^^^^^
+We only need to make one small change to the script: we report to Oríon the objective that we
+want to **minimize** at the end of the script using :py:func:`orion.client.report_results`:
+
+.. literalinclude:: ../../../examples/scikitlearn-iris/main.py
+   :language: python
+   :lines: 32-
+
+In our example, we measure the accuracy of the model to qualify its performance. To get the best
+accuracy possible, we need to minimize the difference between 1 and the accuracy to get it as
+close to 1 as possible. Otherwise, we'll be minimizing the accuracy which will yield a poor model.
+
+:py:func:`orion.client.report_results` can be imported using :
+
+.. code-block:: python
+
+   from orion.client import report_results
+
+Updating the script call
+^^^^^^^^^^^^^^^^^^^^^^^^
+The last missing piece in automating the hyper-parameter optimization of our example model is to
+supply Oríon with the values to use for *epsilon*.
+
+We specify the search space in the command line using ``orion~loguniform(1e-5, 1.0)``
+as the argument for *espilon*. This argument will tell Oríon to use a log uniform distribution
+between ``1e-5`` and ``1`` for the values of *epsilon*.
+
+Putting everything together, we need to call ./main.py with Oríon. The syntax is the
+following: ``$ orion hunt python main.py 'orion~loguniform(1e-5, 1.0)'``. Before executing it on
+your terminal, you have to specify the name of the experiment using the ``-n`` option. It is also a
+good idea to specify a stopping condition using ``--max-trials`` otherwise the optimization will
+not stop unless you interrupt it with :kbd:`ctrl-c`:
+
+.. code-block:: bash
+
+   $ orion hunt -n scitkit-iris-tutorial --max-trials 50 ./main.py 'orion~loguniform(1e-5, 1.0)'
+
+.. warning::
+   Make sure you installed the dependencies for the script before running it using ``pip install
+   -r requirements.txt``.
+
+Viewing the results
+-------------------
+Once the optimization reached its stopping condition, you can query Oríon to give you the results
+of the optimization with the sub-command ``$ orion info``:
+
+.. code-block:: bash
+
+   orion info -n scitkit-iris-tutorial
```

Comment:
Maybe point to user/monitoring.html#library-api for examples on how to query the results from the DB?

Reply:
Good idea!

Reply:
Instead of using the url you can reference it with `` :ref:`library-api-results` ``, the anchor already defined for the section here https://github.com/Epistimio/orion/blame/master/docs/src/user/monitoring.rst#L20